### ISSUE TITLE
Divider line isn't changing color along with the text.

### ISSUE THREAD

**Issue by RaulMarquezInclan** [znjameswu/flutter_math#1]
Hi,

The divider line isn't changing color along with the text, it always displays black.

Code:
```
  final String content = r'\left \| \overrightarrow{F}\right \|= K \frac{\left | Q_1\cdot Q_2 \right |}{d^2}';

  var math = FlutterMath.fromTexString(content,
      settings: Settings(
        colorIsTextColor: true,
      ),
      options: Options.displayOptions.withColor(Colors.orange));
```
Output:
![image](https://user-images.githubusercontent.com/29371626/88280937-a47eec00-cc9b-11ea-8bb5-9797ab351b16.png)

Thanks for the plugin, awesome work.

**Comment by znjameswu** [comment]
Thanks for the comment. I'm working on a fix.

**Comment by znjameswu** [comment]
This issue is fixed in version 0.1.4

**Comment by RaulMarquezInclan** [comment]
Awesome! Thanks again.
